style.css

```diff
@@ -75,3 +94,163 @@ header nav {
   list-style-type: none;
   margin-left: 20px;
 }
+
+.cards_container {
+  background-color: white;
+  border-radius: 20px;
+  border: 2px solid rgb(211, 211, 211);
+  padding: 30px;
+  margin: 200px 30px 30px 30px;
+}
```

Comment:
Please ensure that the content of the project is aligned with the `flexbox` according to the project requirement.